src/main/java/seedu/address/model/wedding/Wedding.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.model.wedding;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.model.person.Person;
+
+public class Wedding {
+    //TODO: create class for WeddingName
+    private final String name;
+    private final Set<Person> contacts = new HashSet<>();
+
+    public Wedding(String name, Set<Person> contacts) {
+        this.name = name;
+        this.contacts.addAll(contacts);
+    }
```

Comment:
I personally think there should be another field for the clients (wedding couple), and contacts could be further clarified to be serviceProviders or something along the lines of that because this would make it easier to distinguish different weddings from each other and the access of couple information more fast & straightforward  since you won't always have to filter for clients.

Also, the addition of a weddingDate field and getDate/setDate methods. Besides that, LGTM!!